lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -13,6 +13,10 @@ function canMultiplyLikeTermConstantNodes(node) {
     return false;
   }
 
+  if (args.every(arg => ConstantOrPowerTerm.getExponentNode(arg).value === 1)) {
```

Comment:
@evykassirer not sure I understand, does this change the functionality at all, or is it just a cosmetic change? cc @aliang8